### ISSUE TITLE
Talkback crashes with Netflix

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#161]
```
What steps will reproduce the problem?
1. Open Netflix, and explore the screen, or flick left or right, and Talkback 
will crash.
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
Latest Talkback from Google play store, and Jellybean 4.1.1

Please provide any additional information below.


```

Original issue reported on code.google.com by `wessmith...@gmail.com` on 30 Nov 2012 at 9:44

**Comment by GoogleCodeExporter** [comment]
```
There's no crash happening here.  It appears that the Netflix application 
itself is muting and locking the media audio stream that text to speech uses.  
The lock on the stream is not released until the Netflix application is 
terminated.

TalkBack does not have the ability to override this behavior, so this should be 
reported to Netflix.
```

Original comment by `caseybur...@google.com` on 30 Nov 2012 at 10:06
- Changed state: **Invalid**
